### PR TITLE
AB#615 Fixed style and font in map panel

### PR DESCRIPTION
**Pull request by wang-xiaotian**
Using FONT_FAMILY "Arial"; display room name in normal spacing

### CODE REVIEW

File: src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
     public static final int UNIT_SIZE = 8; // how big each block/unit; player movement length also
     public static final int PANEL_WIDTH = 600; // panel size
     public static final int PANEL_HEIGHT = 500;
-    public static final Font MAP_TEXT_FONT = new Font("TimesRoman", Font.PLAIN, UNIT_SIZE*2/3);
+    public static final Font MAP_TEXT_FONT = new Font(FONT_FAMILY, Font.PLAIN, UNIT_SIZE*2/3);
```
Comment (chaynsaw) [inline src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java:40]: How does "Arial" look as opposed to "Monospaced"?

Comment (wang-xiaotian) [reply src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java:40]: Are you asking why the current room name is not spaced out? Each letter of the room's name was printed on a single grid block. Now it is printed as a plaint text starting from a specific grid block.

Comment (wang-xiaotian) [reply src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java:40]: I searched Arial vs Monospaced. seems like they reference the same style "Arial Monospaced" :)

Comment (chaynsaw) [reply src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java:40]: No, I mean, you're using Arial now, but I'm using Monospaced in my PR. Which one do you think looks better? 

On Mac it is in my opinion, noticeably better, but Raymond reports on his Windows machine that while Arial spacing looks weird, it looks sharper than Monospaced, which seems a bit blurry.

Comment (chaynsaw) [reply src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java:40]: I merged my PR. You decide which font to use - both are fine, I think... and the group doesn't seem to care much either, lol.

**Review by chaynsaw — APPROVED** [review]

**Pull request merged**